### PR TITLE
Add chown command for hostpath registry

### PR DESCRIPTION
**Pull request by aweiteka**

### CODE REVIEW

File: install_config/install/docker_registry.adoc [hunk 1]
```diff
@@ -165,6 +166,10 @@ $ oadm registry --service-account=registry \
 ----
 endif::[]
 
+[IMPORTANT]
+====
+The docker registry pod run as user *1001*. This user needs to be able to write to the host directory. You may need to change directory ownership to user ID *1001* with this command: `sudo chown 1001:root <path>`
```
Comment (adellape) [inline install_config/install/docker_registry.adoc]: s/docker/Docker/

s/run/runs/

s/needs to/must/

Try putting the command in a code block (with a `$` prompt):

```
with this command:

----
$ sudo chown 1001:root <path>
----
====
```

**Comment by aweiteka** [comment]
@adellape updated.

**Comment by adellape** [comment]
:+1:

**Comment by adellape** [comment]
[rev_history]
|link:../install_config/install/docker_registry.html[Deploying a Docker Registry]
|Added a Note box about writing to the host directory in the link:../install_config/install/docker_registry.html#storage-for-the-registry[Storage for the Registry] section.
%

**Pull request merged**